docs/src/main/sphinx/release/release-409.md

```diff
@@ -0,0 +1,72 @@
+# Release 409 (1 Mar 2023)
+
+## General
+
+* Add support for dropping fields with a `DROP COLUMN` statement. ({issue}`15975`)
+* Remove the `distributed-index-joins-enabled` configuration property and
+  related functionality. ({issue}`15375`)
+* Fix failure when using non-comparable and non-sortable fields as part of a
+  `VALUES` expression. ({issue}`)
+
+## BigQuery connector
+
+* Fix failure when non-lowercase column names exist in the `query` table
+  function. ({issue}`16075`)
+
+## ClickHouse connector
+
+* Remove support for the `ru.yandex.clickhouse.ClickHouseDriver` legacy JDBC
+  driver. ({issue}`16188`)
+* Remove support for specifying expressions in the `sample_by` table property to
+  prevent SQL injection. ({issue}`16261`)
+
+## Delta Lake connector
+
+* Add support for setting default reader and writer versions using the
+  `delta.default-reader-version` and `delta.default-writer-version`
+  configuration properties. ({issue}`16208`)
+
+## Hive connector
+
+* Add support for Hadoop `DefaultCodec` to Hive formats. ({issue}`16250`)
+* Add a native CSV file format reader and writer. These can be disabled with the
+  `csv_native_reader_enabled` and `csv_native_writer_enabled` session properties
+  or the `csv.native-reader.enabled` and `csv.native-writer.enabled`
+  configuration properties. ({issue}`15918`)
```

Comment:
are these documented?

Reply:
They are not, same goes for all the ones below. Planning on following up on that.